docs/.vuepress/config.js

```diff
@@ -370,6 +370,34 @@ Sitemap: https://kuma.io/sitemap.xml
+            `/direct-access-services /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-direct-access-services`,
+            `/virtual-probes /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-virtual-probes`,
+            `/virtual-probes-port /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-virtual-probes-port`,
+            `/sidecar-env-vars /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-sidecar-env-vars`,
+            `/container-patches /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-container-patches`,
+            `prometheus.metrics.kuma.io/port /docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-port`,
+            `prometheus.metrics.kuma.io/path /docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-path`,
+            `/builtindns /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-builtindns`,
+            `/builtindnsport /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-builtindnsport`,
+            `/ignore /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-ignore`,
+            `traffic.kuma.io/exclude-inbound-ports /docs/${versions.latestMinor}/reference/kubernetes-annotations/#traffic-kuma-io-exclude-inbound-ports`,
+            `traffic.kuma.io/exclude-outbound-ports /docs/${versions.latestMinor}/reference/kubernetes-annotations/#traffic-kuma-io-exclude-outbound-ports`,
+            `/transparent-proxying-experimental-engine /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-transparent-proxying-experimental-engine`,
+            `/envoy-admin-port /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-envoy-admin-port`,
+            `/service-account-token-volume /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-service-account-token-volume`,
+            `/transparent-proxying-reachable-services /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-transparent-proxying-reachable-services`,
+            `prometheus.metrics.kuma.io/aggregate-<name>-enabled /docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-aggregate-name-enabled`,
+            `prometheus.metrics.kuma.io/aggregate-<name>-path /docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-aggregate-name-path`,
+            `prometheus.metrics.kuma.io/aggregate-<name>-port /docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-aggregate-name-port`,
+            `/transparent-proxying-inbound-v6-port /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-transparent-proxying-inbound-v6-port`,
+            `/sidecar-drain-time /docs/${versions.latestMinor}/reference/kubernetes-annotations/#kuma-io-sidecar-drain-time`,
```

Comment:
**Suggestion**: This is the right place now, but I’d like to suggest the following improvement: instead of defining the redirects inline right here, reference an array like so:

```suggestion
            ...kumaIoTagRedirects,
```

and then above (beneath where the `policyRedirects` array is defined), you add the following definition. This saves some repetition of common path segments and is easier to follow.

```javascript
          // Redirects known “kuma.io/*” tags to their respective page in the docs.
          const kumaIoTagRedirects = [
            '/sidecar-injection', '#labels',
            '/mesh', '#kuma-io-mesh',
            '/sidecar-injection', '#kuma-io-sidecar-injection-2',
            '/gateway', '#kuma-io-gateway',
            '/ingress', '#kuma-io-ingress',
            '/ingress-public-address', '#kuma-io-ingress-public-address',
            '/ingress-public-port', '#kuma-io-ingress-public-port',
            '/direct-access-services', '#kuma-io-direct-access-services',
            '/virtual-probes', '#kuma-io-virtual-probes',
            '/virtual-probes-port', '#kuma-io-virtual-probes-port',
            '/sidecar-env-vars', '#kuma-io-sidecar-env-vars',
            '/container-patches', '#kuma-io-container-patches',
            'prometheus.metrics.kuma.io/port', '#prometheus-metrics-kuma-io-port',
            'prometheus.metrics.kuma.io/path', '#prometheus-metrics-kuma-io-path',
            '/builtindns', '#kuma-io-builtindns',
            '/builtindnsport', '#kuma-io-builtindnsport',
            '/ignore', '#kuma-io-ignore',
            'traffic.kuma.io/exclude-inbound-ports', '#traffic-kuma-io-exclude-inbound-ports',
            'traffic.kuma.io/exclude-outbound-ports', '#traffic-kuma-io-exclude-outbound-ports',
            '/transparent-proxying-experimental-engine', '#kuma-io-transparent-proxying-experimental-engine',
            '/envoy-admin-port', '#kuma-io-envoy-admin-port',
            '/service-account-token-volume', '#kuma-io-service-account-token-volume',
            '/transparent-proxying-reachable-services', '#kuma-io-transparent-proxying-reachable-services',
            'prometheus.metrics.kuma.io/aggregate-<name>-enabled', '#prometheus-metrics-kuma-io-aggregate-name-enabled',
            'prometheus.metrics.kuma.io/aggregate-<name>-path', '#prometheus-metrics-kuma-io-aggregate-name-path',
            'prometheus.metrics.kuma.io/aggregate-<name>-port', '#prometheus-metrics-kuma-io-aggregate-name-port',
            '/transparent-proxying-inbound-v6-port', '#kuma-io-transparent-proxying-inbound-v6-port',
            '/sidecar-drain-time', '#kuma-io-sidecar-drain-time',
          ].map(([sourcePath, tagValue]) => `${sourcePath}/ /docs/${versions.latestMinor}/reference/kubernetes-annotations/${tagValue}/ 301`)
```

Reply:
Alright, @kleinfreund I applied the changes, and thank you for the guidance!

Reply:
@MustkimKhatik Thanks! Now you’re just missing `kumaIoTagRedirects` in the list of redirects that are actually applied (e.g. `...kumaIoTagRedirects,` beneath `...policyRedirects,`).

Reply:
Yes, it's done @kleinfreund!

Reply:
Thank you. That’s it. I’ll talk to folks and see whether we can merge this already or whether we want to wait and come up with a sensible strategy for keeping this list of redirects up-to-date with the annotations in Kuma (see https://github.com/kumahq/kuma/blob/master/pkg/plugins/runtime/k8s/metadata/annotations.go#L13).

Reply:
Okay sure @kleinfreund

Reply:
> you won’t just be able to type prometheus.metrics.kuma.io/path into the address bar and find the right docs via redirects because prometheus.metrics.kuma.io isn’t actually a served origin

It's not possible to redirect all subdomains in netlify but we can redirect them one by one: https://answers.netlify.com/t/redirect-all-subdomains-to-main-domain/4804 & https://docs.netlify.com/routing/redirects/ - I'd keep those entries and redirect all subdomains that we know.

Reply:
If we really want to extend this to subdomains, I suggest a second array `kumaIoSubdomainRedirects` because the redirect would be constructed in a different manner:

```javascript
const kumaIoSubdomainRedirects = [
  ['https://prometheus.metrics.kuma.io/port', `/docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-port`],
  ['https://prometheus.metrics.kuma.io/path', `/docs/${versions.latestMinor}/reference/kubernetes-annotations/#prometheus-metrics-kuma-io-path`],
  ['https://traffic.kuma.io/exclude-inbound-ports', `/docs/${versions.latestMinor}/reference/kubernetes-annotations/#traffic-kuma-io-exclude-inbound-ports`],
  ['https://traffic.kuma.io/exclude-outbound-ports', `/docs/${versions.latestMinor}/reference/kubernetes-annotations/#traffic-kuma-io-exclude-outbound-ports`],
].map(([sourceUrl, destinationPath]) => `${sourceUrl} ${destinationPath}/ 301`)
```

I’m leaving those entries out that that have dynamic dynamic segments (e.g. `^prometheus.metrics.kuma.io/aggregate-([a-zA-Z0-9-]+)-(port|path|enabled)$`).

Reply:
agree 👍

Reply:
I'd simply add this one then

Reply:
Very nice, thank you. I’ll consider this done now. :)